pkg/apis/extensions/v1beta1/types.go

```diff
@@ -34,7 +34,7 @@ type ScaleStatus struct {
 	Replicas int32 `json:"replicas"`
 
 	// label query over pods that should match the replicas count. More info: http://releases.k8s.io/HEAD/docs/user-guide/labels.md#label-selectors
-	Selector map[string]string `json:"selector,omitempty"`
+	Selector *LabelSelector `json:"selector,omitempty"`
```

Comment:
@piosz agreed to the string form here:
https://github.com/kubernetes/kubernetes/issues/18528#issuecomment-173362214

(as part of a larger proposal)